http/server.ts

```diff
@@ -395,8 +399,6 @@ export class Server {
       // allow the connection to handle multiple requests in the case of h2.
       this.#respond(requestEvent, httpConn, connInfo);
     }
-
-    this.#closeHttpConn(httpConn);
   }
```

Comment:
If I do not remove `closeHttpConn` here, this will result in a "Bad Resource" error.
But I'm not familiar with http, so I'm not sure if it's okay to remove it.